lib/matplotlib/axes/_axes.py

```diff
@@ -88,6 +88,26 @@ def _plot_args_replacer(args, data):
                          "multiple plotting calls instead.")
 
 
+class ViolinStatFunc:
+    """
+    The :class:`ViolinStatFunc` contains:
+    1) a callable whose first argument is compulsory and is a 1-d list of data
+    that is used to plot the violin. This first argument is not required to be
+    specified.
+    2) an alias for this callable. When violinplot outputs the dictionary of
+    artists, this alias is used to identify the artist object corresponding to
+    this callable
+    3) a list of additional arguments. This list does not contain the
+    aforementioned compulsory 1-d list of data.
+    """
+    def __init__(self, func_callable, **kargs):
```

Comment:
using "kwargs" instead of "kargs" would be more consistent with the rest of the MPL code base